GTG/gtk/backends_dialog.ui

```diff
@@ -10,21 +11,21 @@
   <object class="GtkWindow" id="backends_dialog">
     <property name="can_focus">False</property>
     <property name="window_position">mouse</property>
-    <signal name="delete_event" handler="on_BackendsDialog_delete_event"/>
+    <signal name="delete-event" handler="on_BackendsDialog_delete_event" swapped="no"/>
     <child>
       <object class="GtkAlignment" id="alignment1">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="top_padding">10</property>
-        <property name="bottom_padding">10</property>
-        <property name="left_padding">10</property>
-        <property name="right_padding">10</property>
+        <property name="top_padding">18</property>
+        <property name="bottom_padding">18</property>
+        <property name="left_padding">18</property>
+        <property name="right_padding">18</property>
         <child>
           <object class="GtkBox" id="vbox1">
```

Comment:
what is vbox1? Maybe a better name for this, this file is out of my scope in my current activities, it would be good if you'd take care of it :blush: 
(sorry for that :/ )